library/ecp_curves.c

```diff
@@ -5484,31 +5487,34 @@ static int ecp_mod_p448(mbedtls_mpi *N)
 MBEDTLS_STATIC_TESTABLE
 int mbedtls_ecp_mod_p448(mbedtls_mpi_uint *X, size_t X_limbs)
 {
-    size_t i;
+    size_t round;
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
 
     if (X_limbs <= P448_WIDTH) {
         return 0;
```

Comment:
If the output buffer is too small that is an error.
```suggestion
        return MBEDTLS_ERR_ECP_BUFFER_TOO_SMALL;
```

Reply:
Ah. this is copied from the initial implementation, and I thought it meant not an error, no reduction is possible. I will try and get some clarification on this.